src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,77 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { Oval } from 'react-loader-spinner';
+import styles from './Loader.module.css';
+
+export type Props = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+
+  /**
+   * Layout size of the loader. This affects the overall size and associated
+   * stroke width.
+   */
+  size?: 'sm' | 'md' | 'lg';
+
+  /**
+   * Aria label of the oval. Default is oval-loading. Will be overridden if ariaLabel is passed in props
+   * (Shadowed from OvalProps)
+   *
+   * @see https://mhnpd.github.io/react-loader-spinner/docs/components/oval
+   */
+  ariaLabel?: string;
+
+  /**
+   * Whether the oval is visible. Default is true.
+   * (Shadowed from OvalProps)
+   *
+   * @see https://mhnpd.github.io/react-loader-spinner/docs/components/oval
+   */
+  visible?: boolean;
+};
+
+// Pixel sizes corresponding to EDS size units (pixel = rem * 16)
+const loaderSize = {
+  sm: 16, // --eds-size-2
+  md: 40, // --eds-size-5
+  lg: 80, // --eds-size-10
+};
+
+// Given a loader size, the stroke widths can change
+const loaderStrokeSize = {
+  sm: 1,
+  md: 2,
+  lg: 3,
+};
+
+/**
+ * BETA: This component is still a work in progress and is subject to change.
+ *
+ * `import {Loader} from "@chanzuckerberg/eds";`
+ *
+ * Show a loading state within a component container, or a specific page or area, where
+ * arbitrary content will appear once some request has completed.
+ */
+export const Loader = ({
```

Comment:
`Loader` seems really vague to me and could mean several things (like a component that fetches data); I'd really prefer something like `LoadingSpinner`.

Reply:
@dierat fair point. I have used `LoadingIndicator` in the past, which is less ambiguous and much less likely to conflict with other stuff.

I'm waiting for a figma before merging this so once we go thru that, I will make the naming update